README.md

```diff
@@ -107,6 +114,13 @@ manually in a production environment.
   ~$ drush make --working-copy --contrib-destination=profiles/ding2/ drupal.make htdocs
 ```
 
+If you want to apply our custom Drupal core robots.txt patch, you still need to
+do that manually, as it's not possible to add it to the standalone drupal.make
+file
+```sh
+  patch -p1 < profiles/ding2/patches/drupal_core.robots.txt.ding2.patch
+```
```

Comment:
I understand the problem referring to a local patch if the user has only retrieved the single make file. I would prefer that we updated our approach for the alternative installation method to clone the entire repository and then run `drush make` on the cloned `drupal.make` file.